src/NodaTime/TimeInterval.cs

```diff
@@ -0,0 +1,284 @@
+// Copyright 2009 The Noda Time Authors. All rights reserved.
+// Use of this source code is governed by the Apache License 2.0,
+// as found in the LICENSE.txt file.
+
+using JetBrains.Annotations;
+using NodaTime.Text;
+using System;
+using NodaTime.Utility;
+
+namespace NodaTime
+{
+	/// <summary>
```

Comment:
Quick drive-by comment before I do the real review: please format the code with spaces instead of tabs, at a four space indent.

Reply:
> 
> 
> Quick drive-by comment before I do the real review: please format the code with spaces instead of tabs, at a four space indent.

It fixed.